plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/calculator-button.tsx

```diff
@@ -15,7 +16,8 @@ export const CalculatorButton = ( {
 	setIsShippingCalculatorOpen,
 }: CalculatorButtonProps ): JSX.Element => {
 	return (
-		<button
+		<Button
+			render={ <span /> }
```

Comment:
Switched this to be a span, missed it in https://github.com/woocommerce/woocommerce/pull/51375
Was needed to fix its styling.